src/python/pcmdi/meanabs_xy.py

```diff
@@ -1,5 +1,5 @@
 import MV2 as MV
```

Comment:
I agree with @durack1 we should not use `import blah as b` anywhere let's just use the real module names as much as possible, others might be reading this source code you might as well help them learning our tools the proper way. Beside this is 17 add characters, I don't think we use `MV2` 17 times in the file so it's a net loss in typing :wink: